README.md

```diff
@@ -0,0 +1,178 @@
+# Open Context for Node.js
+
+This is a simple library for providing Distributed Context Propagation 
```

Comment:
in tracing circles P in DCP usually stands for "propagation", not "problem". If rewording, could say "for solving the Distributed Context Propagation (DCP) problem"